.gitignore

```diff
@@ -11,3 +11,4 @@
 /public/js/location.js.map
 /storage/debugbar
 .phpunit.result.cache
+.idea/*
```

Comment:
@krievley We're still running into your end of line thing here; would you mind adding a newline here?

Reply:
Other than that, this is good to merge; please fix and merge. Thanks!